api/k8s/v1beta4/k8s_helpers.go

```diff
@@ -47,3 +47,60 @@ func (s *API) WaitForCluster(req *WaitForClusterRequest) (*Cluster, error) {
 	}
 	return cluster.(*Cluster), nil
 }
+
+func (s *API) getNodesWhenReady(req *ListNodesRequest, expectedCount uint32) (interface{}, bool, error) {
+	res, err := s.ListNodes(req, scw.WithAllPages())
```

Comment:
As said below, no need to list the nodes :)

Reply:
No more nodes listing after API fix